Purchases/Purchasing/Purchases.swift

```diff
@@ -737,9 +745,6 @@ public extension Purchases {
      *
      * ``Offerings`` will be fetched and cached on instantiation so that, by the time they are needed,
      * your prices are loaded for your purchase flow. Time is money.
-     *
-     * - Parameter completion: A completion block called when offerings are available.
```

Comment:
These didn't exist on `async` methods 🙃 I removed all of them.

Reply:
great catch, a lot of copy/pasta there